### ISSUE TITLE
Not able Clone tasklist, getting error message

### ISSUE THREAD

**Issue by Srinivas37** [projectknife/Projectfork#1672]
Dear All,

We are using Projectfork 4.3.2 stable version, Joomla 3.4.4.

Issue : We are unable to Clone "Tasklist" using "Save As Copy" option.

Explanation:
We have tasklists with few tasks.
We want to make a copy of Tasklist by using "Save As Copy" option. But we are getting below error message while making copy. Attached screenshot for reference.

Error : Save failed with the following error: Please select a project!

Please check and let us know the fix.

![tasklist_cloning_error](https://cloud.githubusercontent.com/assets/6056715/10191622/33108780-6793-11e5-8e5a-79432fbd27fd.png)

In the above screen, system is asking to select the Project. But we don't have any form field to select Project.

Regards,
Srinivas

**Comment by angek** [comment]
@Srinivas37 I've just tested it as well and can confirm the error. Marking this as a bug and will look into it.

**Comment by angek** [comment]
@Srinivas37 Try the following and see if it fixes the problem for you. Once you confirm I'll submit a bug fix.

open the following file: components/com_pftasks/views/tasklistform/tmpl/edit.php
at around line 171 you should see the following code:
```
        echo $this->form->getInput('alias');
        echo $this->form->getInput('created');
        echo $this->form->getInput('id');
        echo $this->form->getInput('asset_id');
        echo $this->form->getInput('elements');
```

change it to this:

```
       if ($this->item->id > 0) {
            echo $this->form->getInput('project_id');
        }
        echo $this->form->getInput('alias');
        echo $this->form->getInput('created');
        echo $this->form->getInput('id');
        echo $this->form->getInput('asset_id');
        echo $this->form->getInput('elements');
```

please test and let me know if it fixes your problem.

**Comment by Srinivas37** [comment]
Thank you @angek for your response.
It's making a copy of Task list but not the tasks (under tasklist). We want to make a copy of Tasklist along with tasks. Please check.

Regards,
Srinivas

**Comment by Srinivas37** [comment]
Any Updates ???

Regards,
Srinivas

**Comment by Srinivas37** [comment]
Hello @angek, Any updates ?

Regards,
Srinivas

**Comment by Srinivas37** [comment]
Any updates ?

**Comment by angek** [comment]
resolved via commit #92f0a393
Copying of associated tasks is not implemented by design.
